plans/network/traffic.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/testground/sdk-go/network"
+	"github.com/testground/sdk-go/run"
+	"github.com/testground/sdk-go/runtime"
+	"github.com/testground/sdk-go/sync"
+)
+
+func makeTest(whitelistTraffic bool) run.TestCaseFn {
```

Comment:
```suggestion
func routingPolicyTest(allowTraffic bool) run.TestCaseFn {
```

Reply:
Maybe even better to just send the actual policy under test, rather than a `bool` that we then infer on.